server_environment/__manifest__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     "name": "server configuration environment files",
-    "version": "9.0.1.2.0",
+    "version": "10.0.1.2.0",
```

Comment:
My only concern would be the numbers accross different versions.

Nothing warrant that version 10.0.2.1.0 has the exact features and fixes as version 14.0.2.1.0
whereas you could imagine using this number to ensure that your module has all previous fixes.

Thus I don't have a strong opinion on that, I agree with @sbidoul that it can be the choice of the maintainer